doc/Type/independent-routines.pod6

```diff
@@ -634,6 +634,34 @@ token. Directives guide the use (if any) of the arguments. When a directive
 other than C<%> is used, it indicates how the next argument passed is to be
 formatted into the string to be created.
 
+=begin head3 Warning
```

Comment:
Have you tested rendering of this? I mean, should it really be begin/end of head3, instead of simple `=head3 Warning` to make a sub-paragraph? Also, "Warning" is maybe not the best subsection title. Any other ideas?

Reply:
Well, it's correct, but certainly unusual...

Reply:
No testing, but I knew better. Suggestions welcome!